src/WebJobs.Extensions.DurableTask/Microsoft.Azure.WebJobs.Extensions.DurableTask.xml

```diff
@@ -22,11 +22,6 @@
             Configuration for the Durable Functions extension.
             </summary>
         </member>
-        <member name="M:Microsoft.Azure.WebJobs.Extensions.DurableTask.DurableTaskExtension.#ctor">

```

Comment:
Let's remove this file from the PR. I think it randomly changes based on the order of the build and I don't want to clutter the history with random changes to this file.

Reply:
Removed.